edxadapt/edxadapt.py

```diff
@@ -128,7 +128,11 @@ def get_course_id(self):
         # NOTE(idegtiarov) drop meaningless part of default course_id to decrease length of id which is stored in
         # the database
         return self.course_id.course.replace('course-v1:', '')
-
+    
+    # Decorate the view in order to support multiple devices e.g. mobile
+    # See: https://openedx.atlassian.net/wiki/display/MA/Course+Blocks+API
```

Comment:
Hi, thanks for pointing it out. It looks like they are changing the structure of their wiki and I can't find any reference to that specific page. I'll dig a bit further. Anyway, this is the commit where they introduced the `multi_device` support: https://github.com/edx/edx-platform/commit/5a44f391b0011b35cf7b9d3f8b6bba8cfe86a7b5. 
They are referring to `MA-977` but I can't find any real reference to that on the wiki.